bin/reth/src/node/mod.rs

```diff
@@ -46,6 +46,11 @@ pub struct Command {
     #[arg(long, value_name = "PATH", verbatim_doc_comment, default_value_t)]
     db: PlatformPath<DbPath>,
 
+    /// The path to the known peers file. Connected peers are
+    /// dumped to this file on node shutdown, and read on startup.
+    #[arg(long, value_name = "FILE", verbatim_doc_comment, default_value_t)]
+    known: PlatformPath<KnownPeersPath>,
```

Comment:
`--known` as argument is a bit weird imo

the name should make it obvious that this is using peers written to disk.

Reply:
Maybe `--peers-file`?

Reply:
that's a good suggestion.

hmm, I guess it would make sense to provide an option for regular peers as well in `NetworkConfig` and then a helper function to read from disk?

And yes, these functions definitely fit better in the network crate.

Reply:
Renamed it to `--peers-file` and moved the cli flags to `NetworkOpts` in `bin/reth/src/lib.rs`. Also added a function to load peers from file in `NetworkConfig`.